wasm/core/gen.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
If we land this, I'd much prefer a python script.

Reply:
Done.